cloud/ansible/full-deployment.yaml

```diff
@@ -70,33 +72,42 @@
         state: present
         key: "{{ lookup('file', 'public_keys/' + item.username) }}"
 
+    - name: Make users passwordless for sudo in group pzsp
+      lineinfile:
+        path: /etc/sudoers
+        state: present
+        regexp: '^%pzsp'
+        line: '%pzsp ALL=(ALL) NOPASSWD: ALL'
+        validate: 'visudo -cf %s'
+
 - name: Clone the project repository
   hosts: public-machine
   become: yes
   vars:
+#    branch: 21-set-up-azure-terraform-for-production-evironment
     branch: main
   tasks:
     - name: Clone the public repository
       ansible.builtin.git:
         repo: https://github.com/mGarbowski/pzsp2-projekt.git
-        dest: /home/azureuser/project-repo
+        dest: ~/project-repo
         version: "{{ branch }}"  # Specify branch, tag, or commit (optional)
-        force: no      # Do not re-clone if the destination exists
+        force: yes      # Do not re-clone if the destination exists
```

Comment:
Czyli w końcu to wymusza klonowanie nawet jeśli już jest sklonowane czy nie?
Bo chyba powinno żeby zawsze brać najnowsze
ale komentarz został bez zmiany